InstaScore/Controller/ResultViewController.swift

```diff
@@ -1,39 +1,39 @@
 import UIKit
 
-
-
 class ResultViewController: UIViewController{
     var scoreManager = ScoreManager()
+    
     var matchesCount = 0
-    var tescik = "ABC"
     
+    var date1 = ""
+    var date2 = ""
     var homeTeam = [""]
```

Comment:
The below options are the proper ways to set default value and declare type of variable inside class

```suggestion
    var homeTeam: [String] = []
```

or

```suggestion
    var homeTeam = [String]()
```